modules/tinymce/CHANGELOG.md

```diff
@@ -1224,7 +1224,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 5.0.0 - 2019-02-04
 
-Full documentation for the version 5 features and changes is available at https://www.tiny.cloud/docs/release-notes/
+Full documentation for the version 5 features and changes is available at https://www.tiny.cloud/docs/tinymce/5/5_0_0-release-notes-overview.html
```

Comment:
```suggestion
Full documentation for the version 5 features and changes is available at https://www.tiny.cloud/docs/tinymce/5/release-notes/release-notes50.html
```
This link 404s/doesn't exist and I believe this suggestion is the correct one.

Reply:
Lee walked me through this, so accepting on Mike's behalf due to time constraints.